tests/percy/index.js

```diff
@@ -9,30 +9,56 @@ const puppeteer = require('puppeteer');
 const percySnapshot = require('@percy/puppeteer');
 const PORT = 5042;
 
-(async ()=> {
-  const server = new HttpServer({
-    dir: 'test-site/public',
-    port: PORT
-  });
-  server.start();
-
-  const browser = await puppeteer.launch();
-  const page = await browser.newPage();
-
+async function defaultSnapshots(page) {
   const standardPageNavigator = new StandardPageNavigator(page, `http://localhost:${PORT}`);
-  const iframePageNavigator = new IframePageNavigator(page, `http://localhost:${PORT}`, 'iframe_test');
-
   const standardCamera = new Camera(percySnapshot, page);
-  const iframeCamera = new Camera(percySnapshot, page, true);
-
   await (new Photographer(standardPageNavigator, standardCamera).captureSnapshots());
+}
+
+async function iframeSnapshots(page) {
+  const iframePageNavigator = new IframePageNavigator(page, `http://localhost:${PORT}`, 'iframe_test');
+  const iframeCamera = new Camera(percySnapshot, page, true);
   await (new Photographer(iframePageNavigator, iframeCamera).captureSnapshots());
+}
 
+async function nonEnglishSnapshots(page) {
+  const standardPageNavigator = new StandardPageNavigator(page, `http://localhost:${PORT}`);
+  const standardCamera = new Camera(percySnapshot, page);
   standardPageNavigator.setCurrentLocale('es');
   standardCamera.setLocale('es');
-  
   await (new MultilangPhotographer(standardPageNavigator, standardCamera, queryConfig.es).captureSnapshots());
+}
+
+async function RtlSnapshots(page) {
+  const standardPageNavigator = new StandardPageNavigator(page, `http://localhost:${PORT}`);
+  const standardCamera = new Camera(percySnapshot, page);
+  standardPageNavigator.setCurrentLocale('ar');
+  standardCamera.setLocale('ar');
+  await (new MultilangPhotographer(standardPageNavigator, standardCamera, queryConfig.ar).captureSnapshots());
+}
+
+async function runPercyTest() {
+  const server = new HttpServer({
+    dir: 'test-site/public',
+    port: PORT
+  });
+  server.start();
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+
+  const snapshotType = process.argv[2];
+  if (snapshotType === 'rtl') {
+    await RtlSnapshots(page);
+  } else if (snapshotType === 'non-english') {
+    await nonEnglishSnapshots(page);
```

Comment:
was a little confused for second since our RTL snapshots are arabic which is non english

Reply:
yea, updated